pkg/gds/store/trtl/trtl.go

```diff
@@ -0,0 +1,92 @@
+package trtl
+
+import (
+	"errors"
+
+	"github.com/rs/zerolog/log"
+	"github.com/trisacrypto/directory/pkg/gds/client"
+	"github.com/trisacrypto/directory/pkg/gds/models/v1"
+	"github.com/trisacrypto/directory/pkg/gds/store/iterator"
+	"github.com/trisacrypto/directory/pkg/trtl/pb/v1"
+	gds "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
+	"google.golang.org/grpc"
+)
+
+// Open a connection to the Trtl database.
+func Open(profile *client.TrtlProfile) (store *Store, err error) {
```

Comment:
I can see profile making sense here, but we will have to load the configuration from the environment on the GDS server; when we get to that point we should discuss if we want to decouple these or not.